src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAXIMUM_TASKS = 100;
+    private static final String DIVIDER = "===================================================";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        printWelcomeMessage();
+        Scanner sc = new Scanner(System.in);
+        String name = sc.nextLine();
+        printUserGreeting(name);
+        manageTasks(sc, name);
+    }
+
+    private static void manageTasks(Scanner sc, String name) {
+        Task[] tasks = new Task[MAXIMUM_TASKS];
+        int taskNumber = 0;
+
+        while (true) {
+            String input = sc.nextLine();
+            String[] parseInput = input.split(" ",2);
+            String command = parseInput[0];
+
+            if (command.equalsIgnoreCase("done")) {
+                printDoneTasks(tasks, parseInput[1]);
+            } else if (command.equalsIgnoreCase("list")) {
+                printList(tasks, taskNumber);
+            } else if (command.equalsIgnoreCase("todo")) {
+                taskNumber++;
+                addToDoTask(tasks, taskNumber, parseInput);
+            } else if (command.equalsIgnoreCase("bye")) {
+                printByeMessage(name);
+                break;
+            } else if (command.equalsIgnoreCase("event")) {
+                taskNumber++;
+                addEvent(tasks, taskNumber, parseInput[1]);
+            } else if (command.equalsIgnoreCase("deadline")){
+                taskNumber++;
+                addDeadline(tasks, taskNumber, parseInput[1]);
+            } else {
+                taskNumber ++;
+                System.out.println(String.format("%50s", "added: " + input));
+                tasks[taskNumber] = new Task(input);
+                System.out.println(DIVIDER);
+            }
+        }
+    }
+
+    private static void printList(Task[] tasks, int taskNumber) {
+        System.out.println(String.format("%50s", "Here are the tasks in your list:"));
+        for (int i = 1; i <= taskNumber; i++) {
+            System.out.println(String.format("%50s", i + ". " + tasks[i]));
+        }
+        System.out.println(DIVIDER);
+    }
+
+    private static void addDeadline(Task[] tasks, int taskNumber, String s) {
+        System.out.println(String.format("%50s", "Got it. I've added this deadline:"));
+        String[] event = s.split("/by",2);
```

Comment:
Maybe could change the variable name to deadline? The variable name is ambiguous as the method is addDeadline